processors/baggage/baggagetrace/README.md

```diff
@@ -0,0 +1,47 @@
+# Baggage Span Processor
```

Comment:
Please follow https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#documentation for documentation and exmaples